modules/sdk-core/src/bitgo/environments.ts

```diff
@@ -112,11 +112,7 @@ const testnetBase: EnvironmentTemplate = {
   polygonscanBaseUrl: 'https://api-testnet.polygonscan.com',
   polygonscanApiToken: process.env.POLYGONSCAN_API_TOKEN,
   // https://monitor.jungletestnet.io/#apiendpoints for more endpoints
-  eosNodeUrls: [
-    'https://jungle3.cryptolions.io:443',
-    'https://jungle3.eosdac.io:443',
-    'https://jungleapi.nodeone.io:443',
-  ],
+  eosNodeUrls: ['https://kylin.eosn.io'],
```

Comment:
might be a good idea to add additional nodes for redundancy: https://github.com/cryptokylin/CryptoKylin-Testnet#http-api-list

(also update comment to reference these endpoints)